pkg/gatherers/clusterconfig/host_subnets.go

```diff
@@ -11,17 +11,32 @@ import (
 	"github.com/openshift/insights-operator/pkg/record"
 )
 
-// GatherHostSubnet collects HostSubnet information
+// GatherHostSubnet Collects HostSubnet information.
```

Comment:
I don't understand this change TBH. Why capital letter? I agree it's a sentence so I would say:
`GatherHostSubnet collects HostSubnet information.`

Reply:
Check how it looks at the document: https://github.com/openshift/insights-operator/blob/8af0f63eb6941ced61b925a59b43ab7038bcf0cf/docs/gathered-data.md

Reply:
Basically, when generating the documentation the name of the gather is a title and the description paragraph starts with `Collect...`

Reply:
Ahh I see. OK thanks.